modules/theme/src/theme-core/definitions/apps/user-portal.less

```diff
@@ -22,6 +22,19 @@
 
 @import (multiple) '../../theme.less';
 .loadVariables();
+/*-----------------------------
```

Comment:
Haing in `login-portal.less` is enough. Since the styles only apply to login pages.

Reply:
Fixed 2fb29485d8588c2690eb42a8ca6460d9854a28f0